atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -229,18 +232,19 @@ private boolean isLatestValueEmpty(Cell cell, PeekingIterator<RowResult<Value>>
         return false;
     }
 
-    private SweepTimestampsAndSentinels getTimestampsToSweep(Cell cell,
-                                                             Collection<Long> startTimestamps,
-                                                             @Modified Map<Long, Long> startTsToCommitTs,
-                                                             long sweepTimestamp,
-                                                             boolean sweepLastCommitted,
-                                                             SweepStrategySweeper strategySweeper) {
-        Set<Long> uncommittedTimestamps = Sets.newHashSet();
-        SortedSet<Long> committedTimestampsToSweep = Sets.newTreeSet();
+    private SweepTimestampsAndSentinels getTimestampsToSweep(
+            Cell cell,
+            Collection<Long> startTimestamps,
+            LoadingCache<Long, Long> startTsToCommitTs,
+            long sweepTimestamp,
+            boolean sweepLastCommitted,
+            StrategySweeper strategySweeper) {
+        Set<Long> uncommittedTimestamps = new HashSet<>();
+        SortedSet<Long> committedTimestampsToSweep = new TreeSet<>();
```

Comment:
The problem with the other indentation is that it changes whenver the return type, method name, etc changes so you end up with massive diffs. indentation of 8 is what is used instead usually.